news/2 Fixes/4538.md

```diff
@@ -0,0 +1 @@
+Add notebook codicon for Juypter viewContainer.
```

Comment:
I know that we used to sometimes update the changelog on each port. But seems easiest to me to just do a final pass on news before release to pick up all the ports. But if we don't want to I can totally update now.

Reply:
I like the opposite actually. In case somebody forgets to update the news entries, the changelog is mostly kept up to date.

Reply:
@rchiodo CL updated. Also seemed like something from joyce was in so I added that as well.

Reply:
so what's the plan, do I also update the change log?